docs/mcpfile.md

```
@@ -0,0 +1,676 @@
+### 2.1. Top-Level Object
+
+The root of the Tool Definitions File has the following structure:
+
+| Field               | Type                        | Description                                                                                                                                                                                                          | Required |
+|---------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| `kind`              | string                      | Must be `"MCPToolDefinitions"`.                                                                                                                                                                                      | Yes      |
+| `schemaVersion`     | string                      | The version of the MCP file format. Must be `"0.2.0"`.                                                                                                                                                               | Yes      |
+| `name`              | string                      | The name of the server.                                                                                                                                                                                              | Yes      |
+| `version`           | string                      | The semantic version of the server's toolset.                                                                                                                                                                        | Yes      |
+| `instructions`      | string                      | A set of instructions provided by the server to the client about how to use the server.                                                                                                                              | No       |
+| `invocationBases`   | object                      | A set of reusable base configurations for invocations. Each key is a unique identifier, and each value is an invocation configuration (either `http` or `cli`). See [Section 7.3](#73-invocation-bases) for details. | No       |
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)